src/book/verdant-sundial/software-information-entropy.ipynb

```diff
@@ -0,0 +1,530 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "def main():\n",
```

Comment:
Does this block have to exist as a function or could it be used without enacapsulation? Consider moving it outside the function if possible and avoiding the use of the `if __name__ == "__main__":` script-like usage.